budget-service/src/test/java/org/java/mentorship/budget/BudgetApplicationTest.java

```diff
@@ -0,0 +1,12 @@
+package org.java.mentorship.budget;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class BudgetApplicationTest {
+
+    @Test
+    void mainShouldStartApplication() {
+        Assertions.assertDoesNotThrow(() -> BudgetApplication.main(new String[]{}));
```

Comment:
This test is already present on the main branch, but appears to be created under this PR
Please do a rebase with the main branch so you get rid of all of these changes, since they're already on main